gapic/samplegen_utils/utils.py

```diff
@@ -28,6 +28,24 @@
 VALID_CONFIG_TYPE = "com.google.api.codegen.SampleConfigProto"
 
 
+def is_valid_sample_cfg(
```

Comment:
nitty detail: I would have a 1-line docstring for consistency

Reply:
Done.